web-registry/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx

```diff
@@ -47,146 +46,128 @@ const ProjectPageMetadata: React.FC<React.PropsWithChildren<MetadataProps>> = ({
               />
-            )}
-            {endDate && (
-              <MetaDetail label="project end date" data={formatDate(endDate)} />
-            )}
+            </Box>
+            <Body size="xl">{onChainProjectId}</Body>
           </Grid>
-        </Box>
-      </Collapse>
-      <ExpandButton
-        sx={{
-          mt: 2,
-          px: [0],
-          ':hover': { bgcolor: 'transparent !important' },
-        }}
-        onClick={() => setExpanded(!expanded)}
-        text="see"
-        size="small"
-        expanded={expanded}
-      />
+          {methodology && (
+            <MetaDetail
+              label="methodology"
+              data={
+                <ArrowLink
+                  label={methodology?.['schema:name']}
+                  href={methodology?.['schema:url'] || ''}
+                />
+              }
+            />
+          )}
+          {approvedMethodologies && methodologyCount && methodologyCount > 0 && (
+            <MetaDetail
+              label={`methodolog${methodologyCount > 1 ? 'ies' : 'y'}`}
+              data={
+                <>
+                  {approvedMethodologies.map(methodology => (
+                    <ArrowLink
+                      label={methodology?.['schema:name']}
+                      href={methodology?.['schema:url'] || ''}
+                    />
+                  ))}
+                </>
+              }
+            />
+          )}
+          {toucanProjectTokenId && (
+            <MetaDetail
+              label="toucan project token id"
+              data={
+                <ArrowLink
+                  label={toucanProjectTokenId.toString()}
+                  href={metadata?.['regen:toucanURI'] || ''}
+                />
+              }
+            />
+          )}
+          {vcsProjectId && (
+            <MetaDetail
+              label="vcs project id"
+              data={
+                <ArrowLink
+                  label={vcsProjectId.toString()}
+                  href={metadata?.['regen:vcsProjectPage'] || ''}
+                />
+              }
+            />
+          )}
+          {projectType && (
+            <MetaDetail
+              label={`${vcsProjectId ? 'vcs ' : ''}project type`}
+              data={projectType}
+            />
+          )}
+          {projectDesignDocument && (
+            <MetaDetail
+              label="documents"
+              data={
+                <ArrowLink
+                  label="Project Design Document"
+                  href={projectDesignDocument}
+                />
+              }
+            />
+          )}
+          {cfcProjectId && (
+            <MetaDetail
+              label="reference id (cfc project id)"
+              data={
+                <ArrowLink
+                  label={cfcProjectId}
+                  href={metadata?.['regen:cfcProjectPage'] || ''}
+                />
+              }
+            />
+          )}
+          {offsetProtocol && (
```

Comment:
this needs to be removed since it already appears in the ProjectTopSection (as the methodology)